lib/osv/ecosystems.py

```diff
@@ -318,6 +348,7 @@ def enumerate_versions(self, package, introduced, fixed, limits=None):
     'NuGet': NuGet(),
     'PyPI': PyPI(),
     'RubyGems': RubyGems(),
+    'Debian': Debian(),
```

Comment:
Oh, one more thing. Since we are splitting by `Debian:<release version>`, we will need to improve the matching logic here. 

`get` below will need to return the right result for e.g. 'Debian:11'. The release version will need to be tied to the resulting object too since that will affect the enumeration.